### ISSUE TITLE
Fix issues from Pagination zoom and window size accessibility review

### ISSUE THREAD

**Issue by rsmithadhoc** [department-of-veterans-affairs/vets-design-system-documentation#2113]
## Description

Correct issue with `va-pagination` found during zoom and window size accessibility review. 

## Details

- **Should:** When zoomed in and on a smaller screen, the *Previous* and *Next* link gets hidden.
    - <img width="400" alt="image" src="https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/100248909/86fc3cef-2819-4b65-9d74-f3e19cf43f37">
    - **Recommendation**: Do not hide the links on smaller screens. 

## Tasks
- [ ] Fix the accessibility issue

## Acceptance Criteria
- [ ] Accessibility issue addressed

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2113?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @nickjg231 @powellkerry @rmessina1010 @rsmithadhoc

**Comment by caw310** [comment]
USWDS design doesn't have previous and next viewable on mobile so we will table this for the time being. Moving to backlog.

**Comment by rsmithadhoc** [comment]
I will close this one as it matches USWDS and users are still able to navigate between pages using the numbered page links.
